src/screens/UserPortal/Settings/Settings.spec.tsx

```diff
@@ -410,3 +410,34 @@ describe('Testing Settings Screen [User Portal]', () => {
     });
   });
 });
+
+it('prevents selecting future dates for birth date', async () => {
+  await act(async () => {
+    render(
+      <MockedProvider addTypename={false} link={link}>
+        <BrowserRouter>
+          <Provider store={store}>
+            <I18nextProvider i18n={i18nForTest}>
+              <Settings />
+            </I18nextProvider>
+          </Provider>
+        </BrowserRouter>
+      </MockedProvider>,
+    );
+  });
+
+  const birthDateInput = screen.getByLabelText(
+    'Birth Date',
+  ) as HTMLInputElement;
+  const today = new Date().toISOString().split('T')[0];
+
+  // Trying future date
+  fireEvent.change(birthDateInput, { target: { value: '2100-01-01' } });
+
+  // Checking if value is not updated to future date
+  expect(birthDateInput.value).not.toBe('2100-01-01');
```

Comment:
Please change to a calculated date in the future. 2100 is a long way away, but you never know!